provider/spark.go

```diff
@@ -250,8 +250,11 @@ type SparkExecutor interface {
 type SparkStore interface {
 	UploadSparkScript() error //initialization function
 	ResourceKey(id ResourceID) (string, error)
+	ResourceKeys(id ResourceID) ([]string, error)
 	ResourceStreamConv(key string, begin int64) (chan interface{}, error)
-	ResourceRowCt(key string) (int, error)
+	ResourceMultiPartStream(id ResourceID) (chan interface{}, error)
```

Comment:
This interface is really confusing, but refactoring is out of scope for this. Will create a task to clean this up after so we can get this working